code/modules/assembly/health.dm

```diff
@@ -36,16 +40,18 @@
 	if(connected && connected.holder)
 		A = connected.holder
 
-	for(A, A && !ismob(A), A=A.loc); //??? wtf
+	for(A, A && !ismob(A), A=A.loc); // For A, check A exists and that its not a mob, if these are both true then set A to A.loc and repeat
```

Comment:
This bit of code is very painful, the closest things I've found to this is `is_found_within()`, but even that doesnt return what we need here. I blame NT Station for creating this terrible code.